pkg/bloombuild/planner/planner.go

```diff
@@ -135,10 +159,9 @@ func (p *Planner) stopping(_ error) error {
 func (p *Planner) running(ctx context.Context) error {
 	go p.trackInflightRequests(ctx)
 
-	// run once at beginning
-	if err := p.runOne(ctx); err != nil {
-		level.Error(p.logger).Log("msg", "bloom build iteration failed for the first time", "err", err)
-	}
+	// run once at beginning, but delay by 1m to allow ring consolidation when running in SSD mode
```

Comment:
I think this ticker could be replaced with a simpler `time.Sleep` at the beginning of the function if the ringWatcher is not null.

Reply:
To me, a sleep is easier to understand as I don't need to think about which select will trigger earlier. But I don't have a strong opinion on this. Approved.